src/headers/enrollment_op.h

```diff
@@ -64,13 +64,13 @@ typedef struct _enrollment_cert_cfg {
  * @brief Strcture that handles all the enrollment configuration
  * */
 typedef struct _enrollment_ctx {
-    w_enrollment_target *target_cfg;        /**> for details @see _enrollment_target_cfg */
-    w_enrollment_cert *cert_cfg;            /**> for details @see _enrollment_cert_cfg */
-    keystore *keys;                         /**> keys structure */
-    SSL *ssl;                               /**> will hold the connection instance with the manager */
-    unsigned int enabled:1;                 /**> enabled / disables auto enrollment */
-    unsigned int allow_localhost:1;         /**> 1 by default if this flag is in 0 using agent_name "localhost" will not be allowed */
-    unsigned int delay_after_enrollment:30; /**> 20 by default, number of seconds to wait for enrollment */
+    w_enrollment_target *target_cfg;                                        /**> for details @see _enrollment_target_cfg */
+    w_enrollment_cert *cert_cfg;                                                /**> for details @see _enrollment_cert_cfg */
+    keystore *keys;                                                                        /**> keys structure */
+    SSL *ssl;                                                                                     /**> will hold the connection instance with the manager */
+    unsigned int enabled:1;                                                          /**> enabled / disables auto enrollment */
```

Comment:
Please, chenge `enabled` and `allow_localhost` to `bool`.

Reply:
Fixed. What discouraged me to implement the change at first was seeing that the bitfield usage was spread around the source code. I also think we should use booleans for these cases, but was expecting some downside of the change.